cmake/external_mkldnn.cmake

```diff
@@ -18,11 +18,11 @@ include(ExternalProject)
 
 # Includes blas 3.8.0 in mkldnn
 set(NGRAPH_MKLDNN_SHORT_VERSION 0)
-set(NGRAPH_MKLDNN_FULL_VERSION 0.20)
-set(NGRAPH_MKLDNN_ASSET_VERSION "v0.20")
-set(NGRAPH_MKLDNN_VERSION "v0.20.2")
+set(NGRAPH_MKLDNN_FULL_VERSION 0.21)
+set(NGRAPH_MKLDNN_ASSET_VERSION "v0.21")
+set(NGRAPH_MKLDNN_VERSION "v0.21")
 set(NGRAPH_MKLDNN_MKLML_VERSION "2019.0.5.20190502")
```

Comment:
MKL-DNN 0.21 GitHub asset for Windows has mklml_win_2020.0.20190813.zip
mklml_win_2020.0.20190813 addresses the defect with MKL that reproduces when the library loaded and unloaded from different threads. This defect affects ONNX Runtime and does not impact Tensorflow.
Need windows to pickup that version.